src/main/thrift/parquet.thrift

```diff
@@ -753,6 +820,12 @@ struct ColumnMetaData {
 
   /** Byte offset from beginning of file to Bloom filter data. **/
   14: optional i64 bloom_filter_offset;
+  /** 
+   * Optional statistics to help estimate total memory when converted to in memory
+   * representations. The histogram contained on these statistics can also be useful
+   * in some cases for more fine-grained nullability/list length filter pushdown.
+   */ 
+  15: optional SizeEstimationStatistics size_estimate_statistics;
```

Comment:
```suggestion
  16: optional SizeEstimationStatistics size_estimate_statistics;
```

Reply:
This patch probably needs to rebase before merging.

Reply:
rebased.